image_enc_test.go

```diff
@@ -159,29 +155,26 @@ func TestImageEncryption(t *testing.T) {
 	if _, err := s.Create(ctx, image); err != nil {
 		t.Fatalf("Unable to create image: %v", err)
 	}
-	// Force deletion of lease early to check for proper referencing
-	ls.Delete(ctx, l, leases.SynchronousDelete)
 
 	cc = &encconfig.CryptoConfig{
 		DecryptConfig: &encconfig.DecryptConfig{
 			Parameters: dcparameters,
 		},
 	}
+	// Clean up function cancels lease before deleting the image so the images are
+	// properly deleted
+	defer func() {
+		done(ctx)
+		client.ImageService().Delete(ctx, imageName, images.SynchronousDelete())
```

Comment:
I would definitely encourage using test specific image names here. Also when doing a synchronous delete of 2 images, just add the flag on the second one.